xml/installation-installation-ses_integration.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<section xml:id="ses.integration"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&ses; Integration</title>
+ <para>
+  The current version of &productname; supports integration with
+  &ses;. Integrating &ses; enables &ceph; block storage as well as object
+  and image storage services in &productname;.
+ </para>
+  <section xml:id="ses.installation">
+  <title>Enabling &ses; Integration</title>
+  <para>
+   The &ses; integration is provided through the
+   <package>ardana-extensions-ses</package> RPM package. As this package is
```

Comment:
I remembered we changed the package name to `ardana-ses`, to avoid using the extension term.

Reply:
Fixed. Thanks!